src/Store/index.js

```diff
@@ -28,6 +25,21 @@ const reducer = {
   entitiesDetailsReducer: entitiesDetailsReducer({}),
 };
 
+const middlewares = [
+  promiseMiddleware,
+  Pathways.middleware,
+  Recs.middleware,
+  Systems.middleware,
+  Topics.middleware,
+  Acks.middleware,
+  SystemVariety.middleware,
+  notificationsMiddleware({
+    errorTitleKey: ['message'],
+    errorDescriptionKey: ['response.data.detail'],
+  }),
+  ...(process.env.NODE_ENV !== 'production' ? [logger] : []),
```

Comment:
We should use the Entry/DevEntry method. Using the `NODE_ENV` here is too risky.

Reply:
Also, I don't think `process.env` is available in the browser. :)

Reply:
Initially, I tried to go this way. But, I remember entry-dev file is depricated and Compliance [uses](https://github.com/RedHatInsights/compliance-frontend/blob/4e759a201b89e5007b27c8b0c0a59aabd3aa7b04/src/entry-dev.js#L1) it for differentiating prod and dev envs